<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBaseTestWindow</class>
 <widget class="QMainWindow" name="DataBaseTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="userNameBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="userPasswordBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>519</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialized</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="encodedTextField">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="decodedTextField">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="encodeButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decodeButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="encodedTextField_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="decodedTextField_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
